src/main/webapp/assets/css/global-custom.css

```diff
@@ -319,7 +319,7 @@ body{
 }
 
 .navbar-toggler-icon{
-     background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 32 32' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(50,50,93, 1)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 8h24M4 16h24M4 24h24'/%3E%3C/svg%3E");
+    background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 32 32' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(50,50,93, 1)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 8h24M4 16h24M4 24h24'/%3E%3C/svg%3E");
```

Comment:
You can use font awesome icon

Reply:
This is not related to this PR. It's just that little space at the start that I have changed.
Do you want me to change it either way?
In this PR?

Reply:
Ah okay. Let's keep it.